### ISSUE TITLE
Add scss variables

### ISSUE THREAD

**Issue by matmkian** [zavoloklom/material-design-iconic-font#14]
Hi,

First I'd like to thank you for your great work !

It would be great if you can provide scss/less variables for each icon.
We could use it with pseudo element.

```scss
$md-var-arrow-drop-up: '\f296';

.up-button:after {
    content: $md-var-arrow-drop-up;
}
```

**Comment by ArthurClemens** [comment]
I am in favor of this. I would like the option to extend the icons. For FontAwesome I am doing this:

```
.btn-bookmark {
    .extend_icon(fa, fa-var-bookmark);
}

.extend_icon(fa, @name) {
    &:before {
        &:extend(.fa-icon);
        content: @@name;
    }
}
```

This is possible because FA defines a variable for each character code.

**Comment by pine3ree** [comment]
:+1: 
fa variables are very useful, fixed-width class too...

**Comment by ArthurClemens** [comment]
This project looks abandoned.

**Comment by pine3ree** [comment]
i just added a PR for this (sass version): #37 
please download the master repo in my account and test before using it!!!
If you find any error please let me know or issue a PR directly into my repo.
kind regards.

**Comment by zavoloklom** [comment]
Fixed in [2.0 release](https://github.com/zavoloklom/material-design-iconic-font/releases)
